be/src/Project/Services/Identity/Identity.Api/Areas/Accounts/Resources/AccountResources.de.resx

```diff
@@ -207,6 +216,9 @@
   <data name="SignInText" xml:space="preserve">
     <value>Einloggen</value>
   </data>
+  <data name="SmsMarketingApprovalLabel" xml:space="preserve">
+    <value>Ich stimme einer telefonischen Kontaktaufnahme durch ELTAP zu Marketingzwecken zu</value>
```

Comment:
**suggestion:** Word ELTAP couldn't be in text, because this app is not used only by ELTAP